skate/source/includes/_leaderboardget.md

```diff
@@ -0,0 +1,187 @@
+# Leaderboards
+
+## Get the top N scores
+
+```java
+import gamecenter
+
+gamecenter.setKey("myAPIKey");
```

Comment:
We can talk about this when we actually make the java client, but I'd think you'd instantiate a gc object, initializing it with the key. That way if you have another gc object with another key, they won't conflict at all.